doc/quick-start/README.md

```diff
@@ -1,47 +1,24 @@
 # Quick Start
 
```

Comment:
I think the structure should be:

1. Run from an pre-build xline image

    1. xxxx
    2. xxxx

2. Run from xline source code

    1. xxxx
    2. xxxx

Reply:
Currently, our `Build an Xline docker image` is also compiled from scratch and cannot be considered as `pre-built`. After providing support for arm images in the future PR, we can directly pull the appropriate quick-start image from Docker Hub.
In that PR, I will reorganize this structure again.